### PR TITLE
Clickhouse flamegraphs

### PR DESCRIPTION
**Pull request by macobo**
## Changes

This adds the capability to analyze queries via instance status page and get query stats + flamegraphs back.

We can use this to get back information on performance from our customers and to analyze it in cases where we don't have raw access to their clickhouse cluster.

https://user-images.githubusercontent.com/148820/131013442-00f19760-cf01-4fb9-bd4f-761fe16e2e72.mp4

To get it working in multi-host situations a pretty nasty hack is implemented to make it work.

We probably don't want to open this up for anyone. While `readonly` setting is used, this still is something users should use responsibly. @paolodamico do you have ideas how to permission this sort of thing properly?

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] New/changed UI is decent on smartphones (viewport width around 360px)

### CODE REVIEW

**Comment by macobo** [comment]
Added some permissioning logic:
- Only allow read queries + no DDLs
- Only allow admin level users access to this

100% ready to ship now!

File: ee/clickhouse/system_status.py [hunk 1]
```diff
@@ -80,6 +92,8 @@ def get_clickhouse_slow_log() -> List[Dict]:
             FROM system.query_log
             WHERE query_duration_ms > {SLOW_THRESHOLD_MS}
               AND event_time > %(after)s
+              AND query NOT LIKE 'system.query_log'
```
Comment (neilkakkar) [inline ee/clickhouse/system_status.py:95]: Don't you need a % on this line as well? If I understand correctly, right now it's checking that `query != system.query_log`

Comment (macobo) [reply ee/clickhouse/system_status.py:95]: you're 100% correct

Comment (macobo) [reply ee/clickhouse/system_status.py:95]: Will fix soon - does everything else lgty?

Comment (neilkakkar) [reply ee/clickhouse/system_status.py:95]: Sorry, had a call, looking at the rest below now 😋

Comment (neilkakkar) [reply ee/clickhouse/system_status.py:95]: Yep looks good 👍

**Pull request merged**